src/hooks/index.ts

```diff
@@ -41,15 +41,11 @@ export function useIsArgentWallet(): boolean {
 
 export function useInitTransak() {
   const dispatch = useDispatch<AppDispatch>();
-  const initTransak = (
-    account: any,
-    mobileWindowSize: boolean,
-    currency: string,
-  ) => {
+  const initTransak = (account: any, mobileWindowSize: boolean) => {
     const transak = new transakSDK({
       apiKey: '258960cf-1e17-4419-bf7f-77443282f5da', // Your API Key
```

Comment:
Api key should be in an environment file.

Reply:
Added transak api key to env file